source/jormungandr/jormungandr/travelers_profile.py

```diff
@@ -57,12 +63,6 @@ def __init__(self, walking_speed=1.12, bike_speed=3.33, car_speed=11.11, max_dur
 
         self.wheelchair = wheelchair
 
-        self.keolis_type_map = keolis_type_map
-
-    @property
-    def bss_speed(self):
-        return self.bike_speed
-
     def override_params(self, args):
```

Comment:
fallback modes have a default value in the api, so we will never override this parameter